crabs/detection_tracking/inference_model.py

```diff
@@ -1,16 +1,19 @@
 import argparse
 import csv
-import logging
 import os
 from pathlib import Path
+from typing import Any, List, Optional, TextIO, Tuple, Union
 
 import cv2
 import numpy as np
 import torch
 import torchvision.transforms as transforms
```

Comment:
```suggestion
import torchvision.transforms.v2 as transforms
```
From [here](https://pytorch.org/vision/stable/transforms.html#v1-or-v2-which-one-should-i-use)